verified-broadcast/functions/broadcast.js

```diff
@@ -1,3 +1,5 @@
+const { isAuthenticated } = require('@twilio-labs/runtime-helpers').auth;
```

Comment:
Switching the implementation of `isAuthenticated` here requires an update to the `.env` file and the docs in the `README`. Previously this only required a `PASSCODE` to be set in the environment and the username was hardcoded to "admin". Now it needs `AUTH_PASSCODE` and `AUTH_USERNAME`.

Reply:
Great catch! I changed the `.env` and docs to match the new environment variables.